examples/usgs-3dep-1/n41w106-1/n41w106-1.json

```diff
@@ -3,98 +3,108 @@
   "stac_version": "1.0.0",
   "id": "n41w106-1",
   "properties": {
-    "start_datetime": "1948-01-01T00:00:00Z",
-    "end_datetime": "2013-12-31T00:00:00Z",
-    "gsd": 30,
-    "proj:epsg": 5498,
-    "proj:shape": [
-      3612,
-      3612
-    ],
+    "proj:epsg": 4269,
     "proj:transform": [
-      0.00027777778,
+      0.00027777777803598015,
       0.0,
-      -106.001666667082,
+      -106.00166666708242,
       0.0,
-      -0.00027777778,
-      41.0016666667842
+      -0.000277777777786999,
+      41.00166666678416
+    ],
+    "proj:shape": [
+      3612,
+      3612
     ],
-    "threedep:region": "n40w110",
-    "datetime": "2015-07-17T00:00:00Z"
+    "start_datetime": "2020-10-03T00:00:00Z",
+    "end_datetime": "2020-10-10T00:00:00Z",
+    "sci:citation": "U.S. Geological Survey, 2022, USGS 1 Arc Second n41w106 20220331, accessed 2022-05-09",
+    "datetime": "2020-10-10T00:00:00Z"
```

Comment:
Reasoning to choose the time period end rather than the beginning for the nominal datetime?

Reply:
Didn't have a strong one, but it felt like I should pick _something_ since it's a flat, (mostly) static dataset where datetime searches shouldn't really happen because there's no depth along that dimension.